assets/javascripts/discourse/templates/admin/plugins-slack.hbs

```diff
@@ -34,8 +45,7 @@
     </tr>
   </table>
 
-  {{!-- TODO translate --}}
-  <p>To modify these settings from slack, set up a slash command using <a href="https://github.com/discourse/discourse-slack-official/blob/master/README.md">these instructions</a></p>
+  {{i18n "slack.command.help"}}
```

Comment:
We need the `<p>` tag here

![screenshot from 2017-03-14 17-14-38](https://cloud.githubusercontent.com/assets/4335742/23893513/b888b58e-08d9-11e7-842b-5fc4128268b3.png)